Java: CWE-798: Query to detect hard-coded SHIRO key (#5907)
github/codeql: java/change-notes/2021-05-24-hardcoded-shiro-key-in-api-call.md

lgtm,codescanning
* The query "Hard-coded credential in API call" (`java/hardcoded-credential-api-call`)

>   This query finds hard-coded cipher key for shiro. This vulnerability can lead to arbitrary code execution.

Reviewer: How does this lead to RCE?

> * The query "Hard-coded credential in API call" (`java/hardcoded-credential-api-call`)
>   This query finds hard-coded cipher key for shiro. This vulnerability can lead to arbitrary code execution.

Reviewer 2: ```suggestion
* The query "Hard-coded credential in API call" (`java/hardcoded-credential-api-call`) can now detect a hard-coded Apache Shiro cipher key.
```
